Ubuntu_14.04/resources/cliConfig/KUBERNETES/init.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+readonly ROOT_DIR="$(dirname "$0")/../../.."
+readonly COMMON_DIR="$ROOT_DIR/resources/common"
+readonly HELPERS_PATH="$COMMON_DIR/_helpers.sh"
+readonly LOGGER_PATH="$COMMON_DIR/_logger.sh"
+
+# shellcheck source=Ubuntu_14.04/resources/common/_helpers.sh
+source "$HELPERS_PATH"
+# shellcheck source=Ubuntu_14.04/resources/common/_logger.sh
+source "$LOGGER_PATH"
+
+export KUBERNETES_KUBECONFIGFILE=""
+export KUBERNETES_INT=""
+
+help() {
+  echo "
+  Usage:
+    $SCRIPT_NAME <resource_name> [scopes]
+  "
+}
+
+check_params() {
+  _log_msg "Checking params"
+
+  KUBERNETES_INT="$(shipctl get_integration_resource "$RESOURCE_NAME")"
+  KUBERNETES_KUBECONFIGFILE="$( echo "$KUBERNETES_INT" | jq -r '.kubeConfigContent' )"
```

Comment:
is this `kubeConfigContent` or `masterKubeConfigContent`?

Reply:
This is `kubeConfigContent`.